moped-editor/src/components/GridTable/Filters.js

```diff
@@ -676,38 +745,34 @@ const Filters = ({
           </Button>
         </Grid>
         <Grid item xs={12} md={1}>
-          {Object.keys(filterParameters).length > 0 && (
```

Comment:
This check wasn't doing anything because the `useEffect` above makes sure that the `filterParameters` object always has at least one key.

Reply:
🧹  thankyou